components/brave_rewards/resources/page/components/settingsPage.tsx

```diff
@@ -291,6 +291,18 @@ class SettingsPage extends React.Component<Props, State> {
             onClick={this.actions.hideRedirectModal}
           />
         )
+      case 'upholdCustomerDueDiligenceRequiredModal':
+        return (
+          <ModalRedirect
+            id={'redirect-modal-uphold-customer-due-diligence-required'}
+            errorText={getLocale('redirectModalUpholdCustomerDueDiligenceRequiredText')}
+            titleText={getLocale('redirectModalUpholdCustomerDueDiligenceRequiredTitle')}
+            learnMore={'https://wallet.uphold.com/customer-due-diligence'}
+            buttonText={getLocale('redirectModalClose')}
+            walletType={walletType}
+            onClick={this.actions.hideRedirectModal}
+          />
+        )
```

Comment:
I think we mentioned this in the meeting today, but any changes made to `components/brave_rewards/resources/page` should be made to `components/brave_rewards/resources/android_page` as well.

Reply:
Yep, just pushed them.
@emerick Would you mind helping me with testing this on Android? I don't have an Android device at hand.

Reply:
Sure, ping me whenever you're ready!